cms/migrations/0026_certificate_index_page_slug.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.15 on 2022-08-23 12:37
+
+import logging
+
+from django.db import migrations
+from wagtail.core.models import Page
+from django.contrib.contenttypes.models import ContentType
+
+log = logging.getLogger(__name__)
+
+
+def fix_certificate_index_page(apps, schema_editor):
+    """
+    Get HomePage and Create or update Certificate Index Page. Fix slug name if already created
+    """
+    from cms import models as cms_models
+    from cms.api import ensure_certificate_index
+
+    home_page = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(cms_models.HomePage), live=True
+    ).first()
+    if home_page:
+        try:
+            home_page.get_specific()
+            ensure_certificate_index()
+        except cms_models.HomePage.DoesNotExist as exc:
+            log.info(
```

Comment:
We should print it as `log.error`. same for the below error message.

Reply:
fixed